cmd/contour/servecontext.go

```diff
@@ -241,14 +241,14 @@ type LeaderElectionConfig struct {
 
 // TimeoutConfig holds various configurable proxy timeout values.
 type TimeoutConfig struct {
-	// ConnectionIdleTimeout defines how long the proxy should wait while there
-	// are no active requests before terminating an HTTP connection. Set
-	// to 0 to disable the timeout.
+	// ConnectionIdleTimeout defines how long the proxy should wait while there are
```

Comment:
Is there anything you could add to help describe the difference between Connection & Stream?

Reply:
I'm not sure I could describe it succinctly, but I could probably find a link to add that explains the concepts in more detail -- maybe this? https://developers.google.com/web/fundamentals/performance/http2#streams_messages_and_frames

Reply:
@stevesloka do you want me to add external links to the HTTP/2 spec (https://tools.ietf.org/html/rfc7540#section-2.2) to define these terms? I don't want to clutter with too many links now that I've added the Envoy ones, but let me know what you think here.

Reply:
I guess I was thinking of ways to describe without going out to links to find out. Mostly for the user-facing docs this applies the most to. 

But for instance ConnectionIdleTimeout is the time for `no active requests` vs StreamIdleTimeout is `no request activity`. It's difficult to explain, but that was my thought. It is good for now probably to get it all merged in.

Reply:
@stevesloka I'm just worried that to effectively define these terms and differentiate between them for someone with no prior knowledge of the protocols, it's going to take a couple of paragraphs, which I'm not sure is desirable here. WDYT?

Reply:
I think for now it's better to get *an* improvement merged, and then iterate on it afterwards.

I think it's fine to have links out to other documentation as well.